packages/pxweb2-ui/src/lib/components/Radio/Radio.stories.tsx

```diff
@@ -162,3 +163,28 @@ export const ModalVisibleLegend: StoryFn<typeof Radio> = () => {
     ></Radio>
   );
 };
+
+export const CheckCircle: StoryFn<typeof Radio> = () => {
+  type RadioOptions = 'option1' | 'option2' | 'option3';
+
+  const [selectedRadio, setSelectedRadio] = useState<RadioOptions>('option1');
```

Comment:
Storybook has it's own way of doing state in stories, we could use that, maybe?

https://storybook.js.org/docs/writing-stories/args#setting-args-from-within-a-story